zaza/utilities/generic.py

```diff
@@ -151,3 +153,208 @@ def get_yaml_config(config_file):
     # the pwd.
     logging.info('Using config %s' % (config_file))
     return yaml.load(open(config_file, 'r').read())
+
+
+def get_process_id_list(unit_name, process_name,
```

Comment:
Just a question: Have you seen https://github.com/openstack-charmers/zaza/blob/f8fdc65e50d8eb6f9e2df260dcac269485bd95fa/zaza/model.py#L670 example usage here https://github.com/openstack-charmers/zaza/blob/f8fdc65e50d8eb6f9e2df260dcac269485bd95fa/zaza/charm_tests/test_utils.py#L168

Reply:
`get_process_id_list` will just output the list of PID(s) for a specific process.
But thanks for pointing me to that function. It will help me to simplify `test_services`.